draggablepanel/src/main/java/com/github/pedrovgs/DraggableView.java

```diff
@@ -784,4 +798,73 @@ private void notifyCloseToLeftListener() {
   public int getDraggedViewHeightPlusMarginTop() {
     return transformer.getMinHeightPlusMargin();
   }
+
+  public void initializeTimers(){
+
+    onMaximizeCountdownTimer = new CountDownTimer(2000, 10) {
+      @Override
+      public void onTick(long l) {
+
+        if (getSecondViewAlpha() > 0.95) {
+          notifyMaximizeToListener();
+          new Handler().post(new Runnable() {
+            @Override
+            public void run() {
+              onMaximizeCountdownTimer.cancel();
+            }
+          });
+        }
+      }
+
+      @Override
+      public void onFinish() {
+
+      }
+    };
```

Comment:
@rencsaridogan this is not the way to go to implement the listener mechanism. You are using a polling mechanism to notify your listener and depending on the device this approach will notify at the second X or X+1. You should notify your listener close to the method where the alpha is modified and remove the usage of this count down timer.